script.module.resolveurl/lib/resolveurl/plugins/govad.py

```diff
@@ -22,8 +22,8 @@
 
 class GoVadResolver(ResolveGeneric):
     name = 'GoVad'
-    domains = ['govad.xyz', 'goveed.autos', 'goveed.boats']
-    pattern = r'(?://|\.)(gov[ae]*d\.(?:xyz|autos|boats))/(?:embed-)?([0-9a-zA-Z-]+)'
+    domains = ['govad.xyz', 'goveed.autos', 'goveed.boats', 'goveed.beauty', 'asd1.goveed.beauty']
```

Comment:
`goveed.beauty` should cover `asd1.goveed.beauty`
can you try an embed without `asd1.goveed.beauty` added to `domains`

Reply:
I tested 'goveed.beauty' not working I must use 'asd1.goveed.beauty' this domain
So I did it like you added it in vidbam